qemu/tests/nic_hotplug.py

```diff
@@ -97,6 +96,20 @@ def ping_hotplug_nic(ip, mac, session, is_linux_guest):
             status, output = session.cmd_output_safe(del_route_cmd)
         return status, output
 
+    def device_add_nic(pci_model="virtio", netdev="hostnet1", device_id="net1"):
```

Comment:
@xiywang Yes, make sense, but you actually pass values to the function when the only one time you call it, so I supposed that the default value here is actually not necessary.